build.ps1

```diff
@@ -377,7 +377,13 @@ if (!$SkipBuild) {
                 }
             }
 
-            Copy-Item "*.dsc.resource.json" $target -Force -ErrorAction Ignore
+            # skip copying WindowsPowerShell resource on non-Windows
+            if (!$IsWindows) {
```

Comment:
nit: could skip the negation and just use `if ($IsWindows)` by flipping the contents of the if/else statement

Reply:
good point, will flip